src/db/schema.js

```diff
@@ -0,0 +1,21 @@
+import mongoose from 'mongoose'
+
+const entrySchema = new mongoose.Schema(
```

Comment:
liker denne endringen, men vil bare nevne at det trengs et migration script for eksisterende data på etlant tidspunkt, tho tror ikke det egt trengs med mindre man skal søke etter gammel data, så foreslår å lage et sånn script når enn det er bruk for den dataen så e d gjort

Reply:
kom og til å tenke på nå, hvis schema skal oppdateres nå så hadde det vært greit å ta vare på struktur til scoren (dvs. grid til Wordle og Connections, hvor mange gule / hvor)
Kunne vært lagret i kun en int eller etlant enkelt format

Reply:
Lagd en issue for d #27